src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -119,6 +151,8 @@ class MockClientStats {
       request_timestamps;
   SeqStatus sequence_status;
 
+  std::vector<InferInput*> recorded_inputs{};
```

Comment:
should probably clear these values down below in Reset()

Reply:
Cleared 👍